CrewChiefV4/ACS/ACSGameStateMapper.cs

```diff
@@ -130,8 +130,9 @@ public GameStateData mapToGameStateData(Object memoryMappedFileStruct, GameState
             }
             acsVehicleInfo playerVehicle = shared.acsChief.vehicle[0];
 
-            Boolean isOnline = shared.acsChief.IsOnline > 0;
+            Boolean isOnline = getNameFromBytes( shared.acsChief.serverName ).Length > 0;
```

Comment:
just checking... the server name byte array will be all nulls when you're offline - not localhost or anything like that?